src/main/kotlin/net/atos/zac/app/exception/ErrorCodes.kt

```diff
@@ -29,3 +29,5 @@ const val ERROR_CODE_REFERENCE_TABLE_IS_IN_USE_BY_ZAAKAFHANDELPARAMETERS =
 const val ERROR_CODE_SYSTEM_REFERENCE_TABLE_CANNOT_BE_DELETED = "msg.error.system.reference.table.cannot.be.deleted"
 const val ERROR_CODE_ZRC_CLIENT = "msg.error.zrc.client.exception"
 const val ERROR_CODE_ZTC_CLIENT = "msg.error.ztc.client.exception"
+const val ERROR_CODE_SMARTDOCUMENTS_CONFIGURATION = "msg.error.smartdocuments.configuration"
```

Comment:
```suggestion
const val ERROR_CODE_SMARTDOCUMENTS_NOT_CONFIGURED = "msg.error.smartdocuments.not.configured"
```

? Minor thing. 'configuration' seems a little vague. ;-)

Reply:
Applied